tests/models/test_models.py

```diff
@@ -135,37 +177,23 @@ def test_scvi(save_path):
     model.get_latent_library_size(adata2, indices=[1, 2, 3])
 
     # test transfer_anndata_setup
-    adata2 = synthetic_iid(run_setup_anndata=False)
-    transfer_anndata_setup(adata, adata2)
+    adata2 = synthetic_iid()
+    model._validate_anndata(adata2)
     model.get_elbo(adata2)
 
     # test automatic transfer_anndata_setup + on a view
     adata = synthetic_iid()
+    SCVI.setup_anndata(
+        adata,
+        batch_key="batch",
+        labels_key="labels",
+    )
     model = SCVI(adata)
-    adata2 = synthetic_iid(run_setup_anndata=False)
+    adata2 = synthetic_iid()
     model.get_elbo(adata2[:10])
 
-    # test that we catch incorrect mappings
-    adata = synthetic_iid()
-    adata2 = synthetic_iid(run_setup_anndata=False)
```

Comment:
change categorical variable here and add back test!

Reply:
not adding back bc no longer applies to new setup (worried about tampering with anndata fields before, but since has been moved into the anndata manager)